man/man8/zpool.8

```diff
@@ -2194,6 +2194,11 @@ output.  See the
 option of
 .Nm zpool Cm iostat
 for complete details.
+.It Fl d
+Display the number of leaf VDEV slow IOs.  This is the number of IOs that
+didn't complete in \fBzio_slow_io_ms\fR milliseconds.  This does not
+necessarily mean the IOs failed to complete, just took an unreasonably
+long amount of time.
```

Comment:
Let's add one more sentence to the end, to explain why someone might care.  How about:

"This may indicate a problem with the underlying storage."